cylc/flow/scripts/clean.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""cylc clean [OPTIONS] ARGS
+
+Remove a stopped workflow from the local scheduler filesystem.
+
+NOTE: this command is intended for workflows installed with `cylc install`. If
+this is run for a workflow that was instead written directly in ~/cylc-run and
+not backed up elsewhere, it will be lost.
```

Comment:
It's just occurred to me, we can use the `.service/source` symlink to find out if the workflow was written directly into `~/cylc-run`. If it is, maybe we could require a `--force` option to do the clean?

Reply:
The cylc-run directory is intended to be volatile so I wouldn't put thought into handling these cases in any special way.

Also note the source symlink will be moving/changing soon.